### PR TITLE
Add TextField component with binding

### PR DESCRIPTION
**Pull request by michalstocki**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -2817,9 +2817,9 @@
       }
     },
     "@uxpin/merge-cli": {
-      "version": "2.6.0",
-      "resolved": "https://registry.npmjs.org/@uxpin/merge-cli/-/merge-cli-2.6.0.tgz",
-      "integrity": "sha512-kjQz7gMZNLYv5wCUdeehEYw4oIFoqzoVtnA5rudBQ3w8RTMhCJI4u/4HFr8VnOHJtKTGc2UjjuHK8sEVV7CPZQ==",
+      "version": "2.6.2",
+      "resolved": "https://registry.uxpin.pl:4873/@uxpin%2fmerge-cli/-/merge-cli-2.6.2.tgz",
```
Comment (devfive) [inline package-lock.json]: I suppose we should use `npmjs.org`

File: package-lock.json [hunk 2]
```diff
@@ -2873,7 +2873,7 @@
     },
     "@uxpin/react-docgen-better-proptypes": {
       "version": "0.1.1",
-      "resolved": "https://registry.npmjs.org/@uxpin/react-docgen-better-proptypes/-/react-docgen-better-proptypes-0.1.1.tgz",
+      "resolved": "https://registry.uxpin.pl:4873/@uxpin%2freact-docgen-better-proptypes/-/react-docgen-better-proptypes-0.1.1.tgz",
```
Comment (devfive) [inline package-lock.json]: same above

File: src/components/TrendLine/TrendLine.js [hunk 3]
```diff
@@ -43,7 +43,6 @@ const defaultProps = {
   autoDrawDuration: 1600,
   autoDrawEasing: 'ease-out',
   // eslint-disable-next-line react/default-props-match-prop-types
```
Comment (devfive) [inline src/components/TrendLine/TrendLine.js]: this lint rule can also be removed

File: yarn.lock [hunk 4]
```diff
@@ -1775,9 +1775,10 @@
     lodash.unescape "4.0.1"
     semver "5.5.0"
 
-"@uxpin/merge-cli@2.4.1":
-  version "2.4.1"
-  resolved "https://registry.npmjs.org/@uxpin/merge-cli/-/merge-cli-2.4.1.tgz#71195fe27f451daa58b8344d6de3b35434010a7b"
+"@uxpin/merge-cli@2.6.1-dev-27471":
+  version "2.6.1-dev-27471"
+  resolved "https://registry.uxpin.pl:4873/@uxpin%2fmerge-cli/-/merge-cli-2.6.1-dev-27471.tgz#cd35f46cab5d760c1ba52bd6cca11b981931e552"
```
Comment (devfive) [inline yarn.lock]: `npmjs.org`

**Review by devfive — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (michalstocki) [reply package-lock.json]: Ech… The global NPM configuration is messing up here

**Comment by michalstocki** [comment]
Replaced with #28

**Pull request closed without merging**